articles/policies/global-limit.md

```diff
@@ -0,0 +1,241 @@
+`POST /logout` | [Error Page](#error-page)
+`GET /mf` | [Error Page](#error-page)
+`POST /mf` | [Error Page](#error-page)
+`POST /oauth/reverse` |  [JSON Error](#json-error) (`access_denied`)
+`POST /oauth/revoke` |  [JSON Error](#json-error) (`access_denied`)
+`POST /state/introspect` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /unblock` | [Error Page](#error-page)
+`POST /unlink` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /user/ssodata` | Text: "Rate limit exceed"
+`GET /users/:id/impersonate` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /users/:id/impersonate` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /v2/logout` | [Error Page](#error-page)
+
+## Exceeding the Rate Limit
+If you exceed the provided rate limit for a given API endpoint, you will receive a response with [HTTP Status Code 429 (Too Many Requests)](http://tools.ietf.org/html/rfc6585#section-4) except for the cases documented above. The response will also contain the [HTTP Response Headers](policies/rate-limits#http-response-headers) which you can refer to for more information on the rate limits applicable to that endpoint.
+
+### Response body
+The response body depends on the endpoint and, in particular, the response the endpoint was meant to render originally. For example, if the response was meant to render JSON, then a JSON error response will be sent.
+
+The particular response per endpoint is stated in [Affected endpoints](#affected-endpoints) section, below you will find a description for each case.
+
+#### Error Page
+This response is sent for endpoints that render HTML content to the end user. When the rate limit is exceeded the [Error Page](https://auth0.com/docs/universal-login/custom-error-pages) will be rendered instead of the usual content.
+
+#### JSON Error
+JSON formatted response containing error code and description.
+The error code depends on the endpoint:
+* `access_denied`: for endpoints associated with OAuth
+* `too_many_requests`: for the rest of endpoints that return JSON.
+
+```json
+{
+  "error": "access_denied or too_many_requests",
+  "error_description": "Global rate limit exceeded",
+  "error_uri": "https://.../... documentation url"
+}
+```
+
+#### XML Error
+This response is redered for endpoints that normally return XML.
+The error code depends on the endpoint:
+- `fed:BadRequest` will be sent for WSFED-related endpoints.
+- `wst:RequestFailed` will be used in for WSTrust-related endpoints.
+
+```xml
+<env:Envelope xmlns:env="http://www.w3.org/2003/05/soap-envelope" ...>
+ <env:Body>
+  <env:Fault>
+   <env:Code>
+     <env:Value>env:Sender</env:Value>
+     <env:Subcode>
+      <env:Value>fed:BadRequest or wst:RequestFailed</env:Value>
+     </env:Subcode>
+   </env:Code>
+   <env:Reason>
+     <env:Text xml:lang="en">Global rate limit exceeded</env:Text>
+   </env:Reason>
+   <env:Detail>
+   </env:Detail>
+  </env:Fault>
+ </env:Body>
+</env:Envelope>
+```
+
+## FAQ
```

Comment:
We try to avoid FAQ sections, so let's turn these into headers and move them to near the top of the page:

> ## Exceeding the rate limit
> 
> For every hour that you exceed the global rate limit, Auth0 will write the following to your logs: **You have reached the global limit for your account**.
> 
> You can search for the log entries using the [Dashboard](${manage_url}#/logs).
>
> ### Reducing the number of calls to Auth0
> 
> When you exceed your rate limits, you'll need to reduce the number of calls you make to Auth0. The specifics of what you need to do depends on your use case, but here are some recommendations with which to begin:
>
> * Cache `/.well-known/*` information: this information does not change frequently, so you can usually cache it to reduce the number of times you need to call Auth0
> * Consider requesting an `id_token` instead of calling `/userinfo` to get information about the user